.gitignore

```diff
@@ -13,6 +13,7 @@ BaBar/TrajGeom/
 BaBar/TrkBase/
 BaBar/difAlgebra/
 .sconsign.dblite
+Stntuple
```

Comment:
STNtuple shouldn't be built inside Offline. That's an issue outside this PR, but this makes it part of Offline configuration.

Reply:
This has been removed.